src/Files.App/UserControls/SidebarControl.xaml.cs

```diff
@@ -936,6 +936,66 @@ private void ResizeElementBorder_ManipulationStarted(object sender, Manipulation
 			dragging = true;
 		}
 
+		private async void LoadShellMenuItems(CommandBarFlyout itemContextMenuFlyout, ContextMenuOptions options)
+		{
+			try
+			{
+				if (options.ShowEmptyRecycleBin)
+				{
+					var emptyRecycleBinItem = itemContextMenuFlyout.SecondaryCommands.FirstOrDefault(x => x is AppBarButton appBarButton && (appBarButton.Tag as string) == "EmptyRecycleBin") as AppBarButton;
+					if (emptyRecycleBinItem is not null)
+					{
+						var binHasItems = RecycleBinHelpers.RecycleBinHasItems();
```

Comment:
Can you use the helper you introduced with #11204 ?

Reply:
I just noticed this is unused, I'll remove it altogether